cogs/streams.py

```diff
@@ -55,6 +58,21 @@ def __init__(self, bot):
             await self.bot.say("Error.")
 
     @commands.command()
+    async def youtube(self, channelid: str):
+        """Checks if youtube stream is online"""
+        channelid = escape_mass_mentions(channelid)
+        online = await self.youtube_online(channelid)
+        if online is True:
+            await self.bot.say("https://gaming.youtube.com/channel/{} "
```

Comment:
Hmm, Irdumb has suggested a command group. But I think that would be to much work for now, The safest best would be to simply rename the command.